models/lorenz_96_tracer_advection/shell_scripts/advance_model.csh

```diff
@@ -0,0 +1,131 @@
+#!/bin/csh
```

Comment:
this can't run right now - maybe it needs a disclaimer at the top about it being inactive?   in the future if we fix async2 mode this could be useful.  but it can't work as-is both because filter doesn't support it and because i don't believe the filenames in this script will match where the data is going to be written. (they'll be in netcdf files with .nc extensions, for example.)  

or this file can simply be removed and added back from git if/when async2 is re-enabled.

Reply:
https://github.com/NCAR/DART/tree/tracer_advection_siparcs